test/conftest.py

```diff
@@ -71,7 +75,7 @@ def __init__(self):
             self.session.verify = CA_ROOT_CERTIFICATE
 
     @staticmethod
-    def get_nginx_proxy_containers() -> List[Container]:
+    def get_nginx_proxy_container() -> Container:
```

Comment:
This method fails the test if the list's length is zero of more than one, so we're not really returning a list.